mmv1/products/workstations/WorkstationCluster.yaml

```diff
@@ -155,6 +155,12 @@ properties:
           The service attachment is created when private endpoint is enabled.
           To access workstations in the cluster, configure access to the managed service using (Private Service Connect)[https://cloud.google.com/vpc/docs/configure-private-service-connect-services].
         output: true
+      - !ruby/object:Api::Type::Array
+        name: 'allowedProjects'
+        item_type: Api::Type::String
+        description: |
+          Additional projects that are allowed to attach to the workstation cluster's service attachment. 
```

Comment:
Recommend "Additional project IDs", since we have an API-side validation that doesn't permit other formats of identifiers.

Reply:
Heya @esu are you referring to the name or the description? I suppose the `name` but your comment seems to be on line 162.

Reply:
Hi! Sorry for any confusion, the comment refers to the description. The idea is to avoid having users wonder if they should put a project ID, number, resource path, etc, when the API will only accept project IDs.

Reply:
Fixed in 23e5488